kedro/extras/datasets/pandas/excel_dataset.py

```diff
@@ -65,6 +65,32 @@ class ExcelDataSet(
         >>> reloaded = data_set.load()
         >>> assert data.equals(reloaded)
 
+    Example adding a catalog entry for multi-sheet excel file with the ``YAML API``:
+
+    .. code-block:: yaml
+
+        >>> trains:
+        >>>   type: pandas.ExcelDataSet
+        >>>   filepath: data/02_intermediate/company/trains.xlsx
+        >>>   load_args:
+        >>>     sheet_name: [Sheet1, Sheet2, Sheet3]
+
+    Example multi-sheet excel file using Python API:
```

Comment:
So (as we discovered working this out together) the important piece of info that user needs to know is that to save a multi-sheet excel the user **doesn't** need to do anything fancy in `save_args`. Would you mind putting a comment explaining that.

Once again - really good work and thanks for the contribution!

Reply:
I've added the comment in f723f0de9e68e788f33f230ca4acc7121ad39219. Thank you!